warehouse/tuf/__init__.py

```diff
@@ -72,3 +101,38 @@ def wait_for_success(server: str, task_id: str):
 
     else:
         raise RSTUFError("RSTUF job failed, please check payload and retry")
+
+
+@tasks.task(ignore_result=True, acks_late=True)
+def update_metadata(request: Request, project_id: UUID):
+    """Update TUF metadata to capture project changes (PEP 458).
+
+    NOTE: PEP 458 says, TUF targets metadata must include path, hash and size of
+    distributions files and simple detail files. In reality, simple detail files
+    are enough, as they already include all relevant distribution file infos.
+    """
+    server = request.registry.settings["tuf.rstuf_api_url"]
+    if not server:
+        return
+
+    project = request.db.query(Project).filter(Project.id == project_id).one()
+
+    # NOTE: We ignore the returned simple detail path with the content hash as
+    # infix. In TUF metadata the project name and hash are listed separately, so
+    # that there is only one entry per target file, even if the content changes.
+    digest, _, size = render_simple_detail(project, request, store=True)
```

Comment:
If we explicitly ignore the simple_detail_path, why should the function continue to return it at all? Does any other caller use it?

Reply:
Removed in the commit `refactor: utils.render_simple_detail`